### PR TITLE
gh-387 adding a spelling exception for sysctl

### PR DESCRIPTION
**Pull request by kquinn1204**
Addressing GH issue 387

### CODE REVIEW

File: .vale/styles/RedHat/Spelling.yml [hunk 1]
```diff
@@ -98,6 +98,8 @@ filters:
   - "[sS]erializer"
   - "[sS]erverless"
   - "[sS]harding"
+  - "[sS]ysctl"
+  - "[sS]ysctls"
```
Comment (michelle-purcell) [inline .vale/styles/RedHat/Spelling.yml:102]: @kquinn1204 - vale-at-red-hat SMEs confirmed previously in the slack channel that we do not need to add the plural form of each term into the rule file. The tool should handle plurals so I think you can remove line 102.

@aireilly - WDYT?

**Review by michelle-purcell — APPROVED** [review]
@kquinn1204 - LGTM :+1: 

See my comment about removing the plural entry. Thanks.

**Comment by kquinn1204** [comment]
@michelle-purcell @aireilly removed the plural. Hopefully OK to merge

**Comment by aireilly** [comment]
LGTM

**Pull request merged**